1

```diff
@@ -0,0 +1 @@
+test.txt
```

Comment:
What is this test.txt ?

Reply:
It should have been removed. It was a file that I was using for tests.